packages/neos-ui-editors/src/EditorEnvelope/style.css

```diff
@@ -18,7 +18,8 @@
 }
 
 .envelope--highlight {
-    outline: 2px solid var(--colors-Success);
+    box-shadow: 0 0 0 2px var(--colors-Warn);
+    border-radius: 2px;
```

Comment:
Is the border radius chosen arbitrarily or was that used somewhere else already?